Makefile

```diff
@@ -29,18 +25,16 @@ endif
 export FSTAR_HOME
 export OCAMLPATH
 
-all: minimal krmllib
-	@OCAML_ERROR_STYLE="short" \
-	 $(OCAMLBUILD) $(EXTRA_TARGETS)
+extra: pre krmllib
+	+ OCAML_ERROR_STYLE="short" \
+	OCAMLPATH="$(OCAMLPATH)$(OCAMLPATH_SEP)$(FSTAR_HOME)/bin" $(MAKE) -C krmllib extra
 
 minimal: src/AutoConfig.ml src/Version.ml
-	@# Workaround Windows bug in OCamlbuild
-	$(shell [ -f Karamel.$(FLAVOR) ] && rm Karamel.$(FLAVOR))
-	@OCAML_ERROR_STYLE="short" $(OCAMLBUILD) $(TARGETS)
-	ln -sf Karamel.$(FLAVOR) krml
+	+ OCAML_ERROR_STYLE="short" $(MAKE) -C src
+	ln -sf src/_build/default/Karamel.exe krml
 
 krmllib: minimal
-	$(MAKE) -C krmllib
+	+$(MAKE) -C krmllib
```

Comment:
From the GNU make manual, "5.7.1 How the MAKE Variable Works":

> Using the MAKE variable has the same effect as using a ‘+’ character at the beginning of the recipe line.

Is there a reason why you added these?

Reply:
Yes, I can remove this redundant `+` for clarity, thanks for pointing out!

Reply:
FYI, f3fac5a38e596852cd3e590c4d34b48deccb806e introduced `+$(MAKE)` for the test rule